zipline/pipeline/factors/factor.py

```diff
@@ -621,6 +627,120 @@ def rank(self, method='ordinal', ascending=True, mask=NotSpecified):
         """
         return Rank(self, method=method, ascending=ascending, mask=mask)
 
+    @expect_types(
+        target_slice=Slice,
+        correlation_length=int,
+        mask=(Filter, NotSpecifiedType),
+    )
+    def rolling_pearsonr(self,
+                         target_slice,
```

Comment:
No I suppose `target` would be ok, since the `target_factor` here is just `self`, so there's no ambiguity as to what `target` is.